olive/passes/onnx/conversion.py

```diff
@@ -345,7 +345,7 @@ def _convert_model_on_device(
         )
 
         # save the model to the output path and return the model
-        output_model_path = ONNXModel.resolve_path(output_model_path)
+        output_model_path = ONNXModel.resolve_path(output_model_path, Path(model.model_path).name)
```

Comment:
this idea of using the input model's file name only works for onnx -> onnx passes. Here, the change is not needed

Reply:
Yes, fix is coming.